pagopa/io-app
# chore: [IOBP-520] Add session token request in wallet payment flow

## Short description
This PR adds the eCommerce session token required by the payment API endpoints. 

## List of changes proposed in this pull request
- Added necessary data in the redux store, actions and sagas to handle the session token
- Added `getOrFetchWalletSessionToken` function, which returns the session token if present in the store or dispatches a request action and waits for the data to be available

## How to test
With the `io-dev-api-server`, checkout [this branch](https://github.com/pagopa/io-dev-api-server/pulls) and start a new payment flow from **Profile > Playground > New Wallet > Payment**. You should be able to complete a payment.
